backend/dataall/modules/shares_base/services/shares_enums.py

```diff
@@ -4,7 +4,6 @@
 class ShareableType(GraphQLEnumMapper):
     Table = 'DatasetTable'
     StorageLocation = 'DatasetStorageLocation'
-    View = 'View'
```

Comment:
remove unused. I know it could have waited for another PR, but it was very very small